torch/nn/modules/flatten.py

```diff
@@ -0,0 +1,32 @@
+from .module import Module
+from ..._jit_internal import weak_module, weak_script_method
+
+
+@weak_module
+class Flatten(Module):
+    r"""
+    Flattens a contiguous range of dims into a tensor. For use with nn.Sequential.
+    Args:
+        start_dim: first dim to flatten
+        end_dim: last dim to flatten
+
+    Shape:
+        - Input: :math:`(N, *dims)
+        - Output: :math:`(N, product of dims)`
```

Comment:
This is still inaccurate...

Reply:
It's not entirely clear to me what's desired here. If we want to be completely accurate we could do `(N, \prod_{i=\text{start\_dim}}^{\text{end\_dim+1}} \text{dims[i]}, *dims[end_dim+1:])`, which renders like
![image](https://user-images.githubusercontent.com/6355099/60920404-f568a180-a24c-11e9-912d-d88cbacbe9b2.png).

I'd also appreciate other suggestions on how to write this.

Reply:
Even mentioning `N` is not quite right. The default arguments works for batched tensors, but it doesn't have to be batched. Since you already mentioned what the two dims are, instead of saying the general case, can you just say that explicitly, for the default arguments, what the input and output shapes are?

Reply:
Sorry I'm not sure I understand your comment. If I were to say "explicitly, for the default arguments, what the input and output shapes are", I'd write what I had originally + a note that this is for the default arguments. Is that correct?